posts/2025-03-18-github-readmes/index.qmd

```diff
@@ -0,0 +1,234 @@
+
+README's can be initialized at the creation of the repository, or added later. They are by default displayed in the code tab of the repository, underneath the files. They are encoded in **Markdown**. 
+
+## profile README
+
+Every personal account can have its own README, which is displayed on top of the profile page. This is a place to share information about you and the work you are doing. 
+
+To initialize the profile README 
+
+- create a repository with a name that matches your GitHub user name
+- make it public
+- add a README
+
+More information on the GitHub documentation pages [GitHub documentation pages](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).
+
+# Dressing up the README
+
+Regardless of which type of README, there are some things that can be used to make the page more informative and break up the text. Most of the below options were developed for profile README's, but they can also be used on all kinds of repositories.
+
+
+
+## 😎 Add emojis
+
+You can use emojis in the readme to clarify points (and even add them to commits, as I have just learned, use cases are to be determined). 
+
+- 🕰 You can point out deadlines
+
+- [📕](https://www.webfx.com/tools/emoji-cheat-sheet/) Link to documentation
+
+- ✏ Or invite them to contribute
+
+Here is a [cheat-sheet](https://www.webfx.com/tools/emoji-cheat-sheet/).
+---
+
+## Display Top Contributors
+
+You can add the top contributors of your repo to the README:
+
+```html
+<a href="https://github.com/NBISweden/Training-Tech-shorts/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=NBISweden/Training-Tech-shorts" />
+</a>
+```
+
+<a href="https://github.com/NBISweden/Training-Tech-shorts/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=NBISweden/Training-Tech-shorts" />
+</a>
+
+---
+
+## License
+
+Make it easy for others to know what they can do with your code and add license information into your readme. Often these are placed at the top of the readme for even easier accessibility. They link to the text of the license (not in your repository, but on the web). 
+
+[![](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+Here are some [markdown license badges](https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba) you can use.
+
+---
+
+## Add images
+
+You can add images to break up the text in your README. 
+
+### Title banner
+
+These you can use for projects, but also for your very own user readme. They are easily generated [here](https://leviarista.github.io/github-profile-header-generator/). 
+
+![](/posts/2025-03-18-github-readmes/github-header-image.png)
+
+
+### Octocat
+
+You can [design your very own octocat](https://myoctocat.com/), or [choose an existing one](https://octodex.github.com/),  and display it in your readme: 
```

Comment:
Can you add code examples of what the markdown should be please to add these things?